pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -811,7 +811,7 @@ func KomposeObjectToServiceConfigGroupMapping(komposeObject *kobject.KomposeObje
 // TranslatePodResource config pod resources
 func TranslatePodResource(service *kobject.ServiceConfig, template *api.PodTemplateSpec) {
 	// Configure the resource limits
-	if service.MemLimit != 0 || service.CPULimit != 0 {
+	if service.MemLimit != 0 || service.CPULimit != 0 || service.DeployLabels["limits.ephemeral-storage"] != "" {
```

Comment:
We should follow https://kompose.io/user-guide/#labels convention.

Maybe instead let's do `kompose.ephemeral-storage.limit` and `kompose.ephemeral-storage.request` 

?